CHANGELOG.md

```diff
@@ -1,3 +1,253 @@
+* [1798](https://github.com/grafana/loki/pull/1798) **adityacs**: Support configurable maximum of the limits parameter
+* [1713](https://github.com/grafana/loki/pull/1713) **adityacs**: Log error message for invalid checksum
+* [1706](https://github.com/grafana/loki/pull/1706) **cyriltovena**: Non-root user docker image for Loki.
+
+#### Logcli
+* [2027](https://github.com/grafana/loki/pull/2027) **pstibrany**: logcli: Query needs to be stored into url.RawQuery, and not url.Path
+* [2000](https://github.com/grafana/loki/pull/2000) **cyriltovena**: Improve URL building in the logcli to strip trailing /.
+* [1922](https://github.com/grafana/loki/pull/1922) **bavarianbidi**: logcli: org-id/tls-skip-verify set via env var
+* [1861](https://github.com/grafana/loki/pull/1861) **yeya24**: Support series API in logcli
+* [1850](https://github.com/grafana/loki/pull/1850) **chrischdi**: BugFix: Fix logcli client to use OrgID in LiveTail
+* [1814](https://github.com/grafana/loki/pull/1814) **cyriltovena**: Logcli remote storage.
+* [1712](https://github.com/grafana/loki/pull/1712) **rfratto**: clarify logcli commands and output
+
+#### Promtail
+* [2069](https://github.com/grafana/loki/pull/2069) **slim-bean**: Promtail: log at debug level when nothing matches the specified path for a file target
+* [2066](https://github.com/grafana/loki/pull/2066) **slim-bean**: Promtail: metrics stage can also count line bytes
+* [2049](https://github.com/grafana/loki/pull/2049) **adityacs**: Fix promtail client default values
+* [2075](https://github.com/grafana/loki/pull/2075) **cyriltovena**: Fixes a panic in dry-run when using external labels.
+* [2026](https://github.com/grafana/loki/pull/2026) **adityacs**: Targets not required in promtail config
+* [2004](https://github.com/grafana/loki/pull/2004) **cyriltovena**: Adds config to disable HTTP and GRPC server in Promtail.
+* [1935](https://github.com/grafana/loki/pull/1935) **cyriltovena**: Support stdin target via flag instead of automatic detection.
+* [1920](https://github.com/grafana/loki/pull/1920) **alexanderGalushka**: feat: tms readiness check bypass implementation
+* [1894](https://github.com/grafana/loki/pull/1894) **cyriltovena**: Fixes possible panic in json pipeline stage.
+* [1865](https://github.com/grafana/loki/pull/1865) **adityacs**: Fix flaky promtail test
+* [1815](https://github.com/grafana/loki/pull/1815) **adityacs**: Log error message when source does not exist in extracted values
+* [1627](https://github.com/grafana/loki/pull/1627) **rfratto**: Proposal: Promtail Push API
+
+#### Docker Driver
+* [2076](https://github.com/grafana/loki/pull/2076) **cyriltovena**: Allows to pass inlined pipeline stages to the docker driver.
+* [2054](https://github.com/grafana/loki/pull/2054) **bkmit**: Docker driver: Allow to provision external pipeline files to plugin
+* [1906](https://github.com/grafana/loki/pull/1906) **cyriltovena**: Add no-file and keep-file log option for docker driver.
```

Comment:
May be we want to talk about this where we explained the configuration improvements. This is nice to know that doxker driver now clean logs

Reply:
done